model/schema/core.yaml

```diff
@@ -293,6 +309,32 @@ slots:
       If set to true, then the related entity is uncertain.
     range: boolean
 
+  free_text:
+    description: >-
+      A free text string that describes some aspect of an entity.
+    range: string
+
+  note_type:
+    description: >-
+      The type of note: e.g., cytology, comment, summary.
+    range: OntologyTerm
```

Comment:
The range of 'note_type' should be 'VocabularyTerm' from the controlledVocabulary.yaml, not 'OntologyTerm'; this also means that we'll need to add to the 'imports' list 'controlledVocabulary'

Reply:
Sorry. Fixed now.